### PR TITLE
Support for scope_id parameter

### PR DESCRIPTION
**Pull request by tshihad**

### CODE REVIEW

File: examples/resources/hpegl_vmaas_transport_zone/resource.tf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+
+data "hpegl_vmaas_transport_zone" "tf_zone" {
+  name         = "nsx_transport_zone_overlay"
+  service_name = "NSX-T"
```
Comment (manjunath-batakurki) [inline examples/resources/hpegl_vmaas_transport_zone/resource.tf:5]: do we really need this field here? are we using it ?

File: examples/data-sources/hpegl_vmaas_transport_zone/data-source.tf [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+
+data "hpegl_vmaas_transport_zone" "tf_zone" {
+  name         = "nsx_transport_zone_overlay"
+  service_name = "NSX-T"
```
Comment (manjunath-batakurki) [inline examples/data-sources/hpegl_vmaas_transport_zone/data-source.tf:5]: do we really need this field here? are we using it ?

File: examples/resources/hpegl_vmaas_transport_zone/resource.tf (hunk 1 above)
Comment (tshihad) [reply examples/resources/hpegl_vmaas_transport_zone/resource.tf:5]: yeah, it is using in network

File: examples/data-sources/hpegl_vmaas_transport_zone/data-source.tf (hunk 2 above)
Comment (manjunath-batakurki) [reply examples/data-sources/hpegl_vmaas_transport_zone/data-source.tf:5]: I meant `service_name = "NSX-T"`

**Review by manjunath-batakurki — APPROVED** [review]
LGTM

**Review by gandharvas — APPROVED** [review]

**Pull request merged**